src/barnetilsyn/BarnetilsynInformasjon.tsx

```diff
@@ -13,13 +12,62 @@ import {
   RoutesBarnetilsyn,
   ERouteBarnetilsyn,
 } from './routing/routesBarnetilsyn';
+import { hentDataFraForrigeBarnetilsynSøknad } from '../utils/søknad';
+import { useContext, useEffect, useState } from 'react';
+import { GjenbrukContext } from '../context/GjenbrukContext';
+import { useSpråkContext } from '../context/SpråkContext';
+import { ForrigeSøknad } from './models/søknad';
+import { KnappLocaleTekstOgNavigate } from '../components/knapper/KnappLocaleTekstOgNavigate';
 
 export const BarnetilsynInformasjon: React.FC<InformasjonProps> = ({
   person,
   harBekreftet,
   settBekreftelse,
 }) => {
+  const [kanGjenbrukeForrigeSøknad, settKanGjenbrukeForrigeSøknad] =
+    useState(false);
+  const { skalGjenbrukeSøknad, settSkalGjenbrukeSøknad } =
+    useContext(GjenbrukContext);
+  const [locale] = useSpråkContext();
+
+  const erForrigeSøknadBesvartPåSammeSpråkSomErValgt = (
+    forrigeSøknad: ForrigeSøknad
+  ) => {
+    return (
+      (forrigeSøknad.sivilstatus.årsakEnslig?.label ===
```

Comment:
ISøknad inneholder locale... Denne kaaaan sendes med fra backend?
Litt hacky det også, men, kanskje mer stabilt? 

Tror en tilsvarende sjekk på  `Språktekster.Fødselsnummer` i apiet er en like god sjekk, kanskje? Dette er tekster som brukes på pdl-verdier som søker ikke fyller ut selv.

Reply:
Men er det ønsket oppførsel? Blir ikke forrige søknad styrende for hvilket språk som gjelder, og ikke det som er valgt av bruker i frontend?

Reply:
https://github.com/navikt/familie-ef-soknad-api/pull/955

Reply:
Tror vi må gå for den første løsningen allikevel. Det viser seg at locale bare har blitt lagret/sendt med til api'et for overgangsstønad :(